Source/Core/Monomorphization.cs

```diff
@@ -181,15 +189,6 @@ private bool IsFinitelyInstantiable()
       return true;
     }
     
-    public override Expr VisitBinderExpr(BinderExpr node)
-    {
-      if (node.TypeParameters.Count > 0)
-      {
-        isMonomorphizable = false;
-      }
-      return base.VisitBinderExpr(node);
-    }
-
     public override Expr VisitNAryExpr(NAryExpr node)
```

Comment:
A function may have a body so could contribute to an expanding type cycle. On the other hand, Select and Store are primitives with no bodies. So I thought it would be safe to omit adding any information about them. Does that make sense?